wikidata_game/blacklisted-properties.json

```diff
@@ -0,0 +1 @@
+["P646", "P345", "P434", "P436", "P1953", "P1447", "P3138", "P2013", "P1954", "P2949", "P2600", "P535", "P480", "P5032", "P2638", "P905", "P5786", "P2346", "P435", "P4342", "P4947", "P2397", "P982", "P1651", "P4159", "P3782", "P6578", "P1004"]
```

Comment:
How did we determine this list? I noticed that P1330 (musicbrainz instrument id) for example is missing. Is that because we didn't have any of these present anyway?

Reply:
I ran a py script to generate it form `default.yml`. I initially wanted to read the yaml directly with php, but apparently php can't do that out of the box and it seemed like too much effort trying to configure locally and on toolforge the proper packages to enable php to do that.

In any case `P1330` doesn't exist in our blacklist on master.

Reply:
Gotcha; cool. Lets get this merged and run. I have a suspicion there will be more we need to blacklist once we start looking but we can always add to this list.